### PR TITLE
MERGE THIS RIGHT FUCKING NOW I DON'T CARE ABOUT TRAVIS

### PR DESCRIPTION
**Pull request by Iamgoofball**
how did I not notice this AAAAAAAAAAA

### CODE REVIEW

**Comment by CorruptComputer** [comment]
Merged #10174

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -665,7 +665,7 @@ var/list/uplink_items = list()
 /datum/uplink_item/implants/microbomb
 	name = "Microbomb Implant"
 	desc = "An implant injected into the body, and later activated either manually or automatically upon death. The more implants inside of you, the higher the explosive power."
-	item = /obj/item/weapon/storage/box/syndie_kit/imp_macrobomb
+	item = /obj/item/weapon/storage/box/syndie_kit/imp_microbomb
```
Comment (Incoming5643) [inline code/datums/uplink_item.dm]: https://www.youtube.com/watch?v=-dyO9SWiY7k

**Comment by Iamgoofball** [comment]
WHY HASN'T THIS BEEN MERGED YET

**Comment by RemieRichards** [comment]
Rules is why.

**Comment by Razharas** [comment]
Travis

**Comment by Xhuis** [comment]
# kek

**Comment by LaharlMontogmmery** [comment]
GOOOOOOOOOOOOOOF

**Comment by LaharlMontogmmery** [comment]
I also secretly wish that this doesn't get merged until the next round on Sybil. Kind of.

It would be the meteor pen all over again

**Comment by Razharas** [comment]
Well i saw enough

**Comment by Xhuis** [comment]
fun = removed

**Comment by LaharlMontogmmery** [comment]
Don't close this

Leave it as a reminder that goof will be forever goof

File: code/datums/uplink_item.dm (hunk 1 above)
Comment (Anonus) [inline code/datums/uplink_item.dm]: ONE LINE CHANGE, TWO ITEM UPLINK

**Pull request merged**